package.json

```diff
@@ -33,46 +38,45 @@
     "react-component",
     "toolbox"
   ],
+  "engines": {
+    "node": ">=7.4.0",
```

Comment:
well, we use `const` keyword. I'm not sure which version starts supporting this, but we can set any one above. Please suggest